client/parser/ast.go

```diff
@@ -736,20 +736,18 @@ func (node ASTNode) Updates() ([]model.ScoreUpdate, error) {
 
 		updates := []model.ScoreUpdate{partDecl}
 
-		if len(node.Children) > 1 {
```

Comment:
@daveyarwood I have to check if I'm correct, but it doesn't seem like this branch is necessary. The parser always generates an `EventSequenceNode` here. This is in contrast to say `CramNode`, which has an optional second `EventSequenceNode` child.